### PR TITLE
add notice about docker cleanup

### PR DESCRIPTION
**Pull request by madebyTimo**
add notice about docker cleanup to prevent the runner running out of storage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ name3=runner-3
 name4=runner-4
 ```
 
+To clean up the docker files on a monthly basis so that the runner doesn't run out of space, you can add the following line to your crontab:
+```
+0 2 1 * * root docker system prune --force --all > /dev/null
```
Comment (delvh) [inline README.md:44]: Why not daily?

Comment (madebyTimo) [reply README.md:44]: Because the cache is very useful to speedup builds

Comment (delvh) [reply README.md:44]: weekly?

Comment (delvh) [reply README.md:44]: biweekly?

Comment (madebyTimo) [reply README.md:44]: monthly is enough

**Review by delvh — APPROVED** [review]

**Pull request merged**